test/testers.jl

```diff
@@ -64,4 +64,19 @@
             rrule_test(first, randn(), (Tuple(randn(4)), CTuple{4}(randn(4)...)))
         end
     end
+
+    @testset "symbol input: fsymtest" begin
+        fsymtest(x, s) = x
```

Comment:
since it's a test, perhaps just use an annotation to make this really clear? (And to check it works when the type is restricted i suppose)
```suggestion
        fsymtest(x, s::Symbol) = x
```